arch/arm/mach-bcm283x/Kconfig

```diff
@@ -196,8 +196,16 @@ config TARGET_RPI_ARM64
 	  the RPi 4 model B, in AArch64 (64-bit) mode.
 	select ARM64
 
+config TARGET_FANCOM_HMI
+	bool "Fancom HMI based on CM4"
+	help
+	  fancom
```

Comment:
Maybe place a more descriptive explanation on the help part?

Reply:
done